### ISSUE TITLE
Advancing and Retreating by word doesn't read the word in JAWS

### ISSUE THREAD

**Issue by Bekreth** [Bekreth/screen_reader_terminal#1]

**Comment by Bekreth** [comment]
https://github.com/Bekreth/jane_cli/issues/38
https://github.com/Bekreth/jane_cli/issues/33
